internal/remoteconfig/remoteconfig.go

```diff
@@ -72,6 +72,9 @@ const (
 	APMTracingCustomTags
 )
 
+// APMTracingEnabled enables APM tracing
+const APMTracingEnabled Capability = 19
```

Comment:
Why 19 explicitly, and not just listed in the constants above as an iota? If it has to be different from the others for some reason, please add a comment describing why.